store/blocks.js

```diff
@@ -121,14 +121,14 @@ export const actions = wrapActionsWithResolvedEpoch({
    * @param hash
    * @returns {Promise<*>}
    */
-  async getGenerationFromHash ({ state, rootGetters: { epoch }, commit }, hash) {
+  async getGenerationFromHash ({ state, rootGetters: { node }, commit }, hash) {
     if (state.hashToHeight[hash]) {
       return state.generations[state.hashToHeight[hash]]
     }
-    const generation = await epoch.api.getGenerationByHash(hash)
+    const generation = await node.api.getGenerationByHash(hash)
     generation.numTransactions = 0
     const height = generation.keyBlock.height
-    const resp = await fetch(process.env.VUE_APP_EPOCH_URL + 'middleware/transactions/interval/' + height + '/' + height)
```

Comment:
we can not use fetch to get data from external source anymore.
it should be something like this:
```js
 const resp = await this.$axios.$get(state.nodeUrl + 'middleware/transactions/interval/' + height + '/' + height)
```

Reply:
updated